src/thenable.ts

```diff
@@ -0,0 +1,5 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+interface Thenable<T> extends PromiseLike<T> { }
```

Comment:
Why do we need this if it's the exact same interface? Shouldn't they be compatible if we just use `PromiseLike`?